Update CHANGELOG.md (#8131)
treeverse/lakeFS: CHANGELOG.md

# Changelog

# v1.33.0

**Note:**
PLEASE READ CAREFULLY BEFORE UPGRADING TO THIS VERSION!!!

> This version introduces deprecation of the ACLs functionality, and might require some users to take action.

Reviewer: ```suggestion
This version removes the built-in ACLs functionality, and might require some users to take action.
```
